ros2_batch_job/windows_batch/__init__.py

```diff
@@ -36,7 +36,7 @@ def show_env(self):
         # Show the env
         self.run(['set'], shell=True)
         # Show what pip has
-        self.run([self.python, '-m', 'pip', 'freeze', '--all'])
+        self.run([self.python, '-m', 'pip', 'list'])
```

Comment:
Again, I'd actually advocate for `freeze --all` over just `list`. What prompted the change?